syntax = "proto3";

option java_package = "ex.grpc";

package test_message;

// Defines the service
service TestMessageService {
    // Function invoked to send the request
    rpc sendRequest (TestMessageRequest) returns (TestMessageResponse) {}
}

// The request message containing requested numbers
message TestMessageRequest {
    int32 a = 1;
    int32 b = 2;
}

// The response message containing response
message TestMessageResponse {
    int32 result = 1;
}